### DISCUSSION TITLE
Restrict "Mark as Answer" for the owners of "Discussion" thread

### DISCUSSION THREAD

**Discussion by PranamBhat** [community/community#3606] [Discussions]
I have noticed that if an user starts the "**Discussion**" and comment something over the thread, he can himself "**Mark as Answer**" for his comment/thread.

**This should be restricted.**

![image](https://user-images.githubusercontent.com/32120033/112439351-0635af80-8d6f-11eb-8f84-6b12d0bbc36e.png)

**Comment by markseuffert** [comment]
I have a different view, I like the self service in GitHub discussions and that there's less to do for maintainers. This is one of the main reasons why I switched from GitHub Issues to GitHub Discussion. Discussions are a place where the community helps each other. If a user wants to mark his/her question as answered, that's fine for me, we collect bugs/todos/plans for product development elsewhere.

**Comment by PranamBhat** [comment]
@alexcnichols  @becca @bailey @evi-liu @michellemerrill Need your help here!

**Reply by alexcnichols** [reply]
Hi @PranamBhat - Thanks for filing the feedback. I can see how being able to make one's own answer could lead to some "gaming" of the system potentially but I've also seen it play out in a healthy way where someone does in fact come back to answer their own question. I'm curious to see what other people using Discussions have to say. Until then, and in future instances like this, no need to mention maintainers here unless there's something that is a moderation concern. GitHub employees are regularly watching and engaging in discussions here as needed already.
